### PR TITLE
test-services: tests for kazoo_services

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

**Comment by fenollp** [comment]
Please don't merge yet: I'm collecting coverage data.

File: applications/crossbar/priv/couchdb/schemas/system_config.services.json [hunk 1]
```diff
@@ -24,6 +28,7 @@
         "scan_rate": {
             "default": 20000,
             "description": "services scan rate",
```
Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/system_config.services.json]: Can you update the descriptions to include what the units are? Or migrate from `scan_rate` to `scan_rate_ms`?

Comment (fenollp) [inline applications/crossbar/priv/couchdb/schemas/system_config.services.json]: I don't see why this wouldn't be another PR.

File: core/kazoo_data/src/kz_datamgr.erl [hunk 2]
```diff
@@ -1167,6 +1167,13 @@ add_required_option({Key, Fun}, {JObj, Options}=Acc) ->
 %%% View Functions
 %%%===================================================================
 
+-ifdef(TEST).
+-define(GET_RESULTS(DbName, DesignId, Options)
+       ,io:format(user, "\n~s:get_results(~p, ~p, ~p)\n", [?MODULE, DbName, DesignId, Options])).
+-else.
+-define(GET_RESULTS(DbName, DesignId, Options), ok).
+-endif.
```
Comment (lazedo) [inline core/kazoo_data/src/kz_datamgr.erl]: remove this

File: core/kazoo_data/src/kz_datamgr.erl [hunk 3]
```diff
@@ -1187,33 +1194,29 @@ get_results(DbName, DesignDoc) ->
     get_results(DbName, DesignDoc, []).
 
 get_results(DbName, DesignDoc, Options) when ?VALID_DBNAME(DbName) ->
+    ?GET_RESULTS(DbName, DesignDoc, Options),
```
Comment (lazedo) [inline core/kazoo_data/src/kz_datamgr.erl]: remove

File: core/kazoo_data/src/kz_datamgr.erl [hunk 4]
```diff
@@ -1187,33 +1194,29 @@ get_results(DbName, DesignDoc) ->
...
     end;
-
 get_results(DbName, DesignDoc, Options) ->
     case maybe_convert_dbname(DbName) of
         {'ok', Db} -> get_results(Db, DesignDoc, Options);
         {'error', _}=E -> E
     end.
 
 get_results_count(DbName, DesignDoc, Options) ->
+    ?GET_RESULTS(DbName, DesignDoc, Options),
```
Comment (lazedo) [inline core/kazoo_data/src/kz_datamgr.erl]: remove

File: core/kazoo_data/src/kz_datamgr.erl (hunk 2 above)
Comment (fenollp) [inline core/kazoo_data/src/kz_datamgr.erl]: why?

**Comment by fenollp** [comment]
Ready :)

File: core/kazoo_data/src/kz_datamgr.erl (hunk 2 above)
Comment (lazedo) [inline core/kazoo_data/src/kz_datamgr.erl]: are we running tests unit tests against couchdb ?
